src/components/flash/flash.stories.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
```

Comment:
No notes in this file.

![image](https://user-images.githubusercontent.com/26925206/53956970-0eade580-40d5-11e9-951f-e5f8f3cf473f.png)

Current demo site:
![image](https://user-images.githubusercontent.com/26925206/53956989-1c636b00-40d5-11e9-81a3-48e986add9cb.png)